src/Adapters/FHIRValueSetAdapter.php

```diff
@@ -17,152 +17,129 @@
+        $repositoryClass = $this->repositoryPath . ucfirst($resource) . 'Repository';
         $repository = new $repositoryClass();
         $repository->setConnection($connection);
         $models = $repository->fetchAll();
 
         // Create the value set and expansion to fill with the model data
         $FHIRValueSet = new FHIRValueSet();
+        $valueSetID = new FHIRString();
+        $valueSetID->setValue('test_id');
+        $FHIRValueSet->setId($valueSetID);
+        $text = new FHIRNarrative();
+        $div = new FHIRString();
+        $div->setValue("<div>\n      <h3>Value Set Contents</h3>\n      <p>This value set contains 3 concepts</p>\n      <table class=\"codes\">\n        <tr>\n          <td>\n            <b>Code</b>\n          </td>\n          <td>\n            <b>System</b>\n          </td>\n          <td>\n            <b>Display</b>\n          </td>\n        </tr>\n        <tr>\n          <td>\n            <a name=\"http---hl7.org-fhir-v2-0136-Y\"> </a>\n            <a href=\"v2/0136/index.html#Y\">Y</a>\n          </td>\n          <td>http://hl7.org/fhir/v2/0136</td>\n          <td>Yes</td>\n        </tr>\n        <tr>\n          <td>\n            <a name=\"http---hl7.org-fhir-v2-0136-N\"> </a>\n            <a href=\"v2/0136/index.html#N\">N</a>\n          </td>\n          <td>http://hl7.org/fhir/v2/0136</td>\n          <td>No</td>\n        </tr>\n        <tr>\n          <td>\n            <a name=\"http---hl7.org-fhir-data-absent-reason-asked\"> </a>\n            <a href=\"valueset-data-absent-reason.html#asked\">asked</a>\n          </td>\n          <td>http://hl7.org/fhir/data-absent-reason</td>\n          <td>Don't know</td>\n        </tr>\n      </table>\n      <h2>Yes/No/Don't Know</h2>\n      <p>For Capturing simple yes-no-don't know answers</p>\n      <p>This value set includes codes from the following code systems:</p>\n      <ul>\n        <li>Import all the codes that are contained in \n          <a href=\"v2/0136/index.html\">http://hl7.org/fhir/ValueSet/v2-0136</a>\n        </li>\n        <li>Include these codes as defined in \n          <a href=\"valueset-data-absent-reason.html\">http://hl7.org/fhir/data-absent-reason</a>\n          <table>\n            <tr>\n              <td>\n                <b>Code</b>\n              </td>\n              <td>\n                <b>Display</b>\n              </td>\n            </tr>\n            <tr>\n              <td>asked</td>\n              <td>Don't know</td>\n              <td>The source human does not know the value.</td>\n            </tr>\n          </table>\n        </li>\n      </ul>\n    </div>");
+        $text->setDiv($div);
+
+        $narrativeStatus = new FHIRString();
+        $narrativeStatus->setValue("generated");
+        $text->setStatus($narrativeStatus);
+        $FHIRValueSet->setText($text);
+
+        $url = new FHIRString();
+        $url->setValue("http://hl7.org/fhir/ValueSet/yesnodontknow");
```

Comment:
These values should come from the ValueSet model. Please add getValueSetUrl() & getValueSetName() & getValueSetDescription() to the ValueSe Model and database:
Example: Provider Value Set
url => [base]/fhir/ValueSet/1 (where 1 is the ID of the value set.)
name => Providers
description => For capturing provider selection on GPOnline client application